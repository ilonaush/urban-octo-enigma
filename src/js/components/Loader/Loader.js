import React from 'react';
import "./Loader.styl"

const Loader = () =>
    <div className='loader'>
        Loading...
    </div>
;

export default Loader;
