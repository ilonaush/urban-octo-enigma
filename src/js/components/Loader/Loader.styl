@import "~js/vars";

.loader
  background-color rgba(255,255,255, .5)
  position absolute
  top 0
  left 0
  display flex
  align-items center
  font-size 50px
  text-align center
  width 100%
  justify-content center
  height 100%
  color $orange
