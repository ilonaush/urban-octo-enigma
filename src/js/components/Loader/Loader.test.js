import React from 'react';
import Loader from './Loader';
import {shallow} from "enzyme";

describe('Loader', () => {
  it('render correctly loader component', () => {
    const LoaderComponent = shallow(<Loader/>);
    expect(LoaderComponent).toMatchSnapshot();
  });
});
