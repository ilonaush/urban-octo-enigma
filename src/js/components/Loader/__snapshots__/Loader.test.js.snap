// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader render correctly loader component 1`] = `
<div
  className="loader"
>
  Loading...
</div>
`;
